auctions/english-auction/.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
What is this?

Reply:
it defines the version of node that this project is guaranteed to work on.  It also allows developers an easy way to configure their node env to match that of the repo.  With the .nvmrc file a dev with nvm installed can simply do
`nvm use` then `nvm install` to get their node env configured correctly for the repo.